android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -107,7 +107,7 @@ class AirshipPlugin : MethodCallHandler {
                             .putOpt("extras", JsonValue.wrap(extras))
                             .apply {
                                 if (message.expirationDateMS != null) {
-                                    putOpt("expiration_date", DateUtils.createIso8601TimeStamp(message.expirationDateMS))
+                                    putOpt("expiration_date", DateUtils.createIso8601TimeStamp(message.expirationDateMS!!))
```

Comment:
This needed? It should of smart checked above

Reply:
yes, it was generating an error